### DISCUSSION TITLE
What do you use OpenMage for?

### DISCUSSION THREAD

**Discussion by empiricompany** [OpenMage/magento-lts#3881] [Polls]
I would like to open this survey to understand how each of us uses OpenMage and consequently what is our vision on development. 
Do you use OpenMage to extend the life of old Magento 1 projects or, like me, do you consider it a valid platform for new projects too?

Personally, my entire business is based on OpenMage which currently remains my ecommerce best solution. I have dozens of projects under my belt with satisfied clients.

I would also like to know how many projects you currently manage with OpenMage and what you expect from the project in the future; maintaining a super backwards compatible LTS or developing new features to keep up with the competition? What are your future plans for these projects?

**Comment by andrewcbi** [comment]
I've manually cherry picking / applying improvements manually to a heavily modified version of Magento/OpenMage.

I haven't come across anything else that does exactly what we need and having customised this platform as extensively as we have, I want to give the system as much of a future as possible (plus the number of hours required to replace it is daunting).

We're not using it to handle freight costing or payments so that does take some of the pressure off those parts of maintaining the system.

**Comment by kiatng** [comment]
I just work on a long running project which comprised of 2 productions of OpenMage. With the continuous improvements from the community, OpenMage is now better than ever. Not moving anywhere.

**Comment by pauldpauld** [comment]
As an **end-user,** I have been using Magento/OpenMage since ver 1.4 to run my very small ecommerce business, buying a few basic extensions (like the Australia Post freight extension) to keep my business running.
I upgraded to OpenMage LTS 19.4.14 and stopped there, it's stable, runs well enough for me and I can keep it going with the main aim of keeping my ecommerce business running and giving me an income.
I subscribe to the discussions here to keep up-to-date with what's happening on/with the platform. I would have upgraded to the newer versions but reading the warnings/comments about possible extension incompatibility (mainly due to php8 I beleive) I am concerned that I would need to find someone to adapt these extensions to the newer versions.
I have no intention of moving to another platform.
I DO look at the security patches and apply ALL of them where I can.
As an end-user I don't have new projects.
When anyone asks, I do suggest OpenMage as an excellent product.

**Comment by fballiano** [comment]
I manage 8-10 small-medium stores (I used to manage 2-4 big ones but sadly the companies were sold and the new owners replatformed).

I said it a million times already but I want from OpenMage what's coming in v21 and much more.
The difference between M1.9.4.5 and latest OM20 is extreme, all the customers that I upgraded are very very happy, so we NEED this kind of push forward, also if it means that some store will need to hire somebody to upgrade (I mean, literally what's a software this big that requieres zero effort?)
